demos/common/cpp/models/src/segmentation_model.cpp

```diff
@@ -69,7 +69,7 @@ void SegmentationModel::prepareInputsOutputs(std::shared_ptr<ov::Model>& model)
     }
 
     ov::preprocess::PrePostProcessor ppp(model);
-    ppp.input().tensor().set_element_type(ov::element::u8).set_layout({"NHWC"});
+    inputTransform.setPrecision(ppp, model->input().get_any_name());
```

Comment:
```suggestion
    inputTransform.setPrecision(ppp, model->input().get_any_name());
    ppp.input().tensor().set_layout("NHWC");
```
You've lost `set_layout({"NHWC"})`. Put it back